moduleframework/tests/static/dockerfile_lint.py

```diff
@@ -54,29 +54,34 @@ def _test_for_env_and_label(self, docker_env, docker_label, env=True):
             label_found = self.dp.get_specific_label(docker_label)
         return label_found
 
+    def _get_msg(self, label):
+        return label + " is missing in Dockerfile."
+
     def test_architecture_in_env_and_label_exists(self):
-        self.assertTrue(self.dp.get_specific_label("architecture"))
+        self.assertTrue(self.dp.get_specific_label("architecture"),msg=self._get_msg("architecture env and label"))
```

Comment:
This checks only the label, not the env var, right? So the message is inaccurate.